.prettierignore

```diff
@@ -1,6 +1,7 @@
 *.ejs
 package.json
 package-lock.json
+config/styleguide/config.js
```

Comment:
Why do we want to avoid this file?

Reply:
on commit, prettier kept removing parens around an arrow function's single arg which then fails the build

Reply:
updating eslint rule instead

Reply:
Ah, forgot we're running eslint on this codebase.